src/CssTreeWalker.js

```diff
@@ -20,6 +20,7 @@ CssTreeWalker.prototype.constructor = CssTreeWalker;
 CssTreeWalker.prototype.beginReading = function () {
   this.ast = rework(this.startingSource)
     .use(this.readPlugin.bind(this));
+  return this;
```

Comment:
I didn't mean to keep this change in. Let me know if you'd like me to remove it.